hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2499,4 +2505,21 @@
       connections.
     </description>
   </property>
+  <property>
+    <name>ozone.csi.default-volume-size</name>
+    <value>1000000000</value>
+    <tag>OZONE, CSI</tag>
+    <description>
+      The default size of the create volumes (if not specified).
+    </description>
+  </property>
+  <property>
+    <name>ozone.csi.socket</name>
+    <value>/tmp/csi.sock</value>
```

Comment:
tmp directory might be emptied by admins and others without knowledge of this file.

Reply:
AFAIK it shouldn't be a problem. This is a socket which is created by the process to listen on and even if it's deleted the socket will work well (until the close).

But I modified it to /var/lib/csi.sock. It can be more meaningful (and maybe more safe If my previous statement is wrong ;-) )